Kickstarter-iOS/Views/ReferralChartView.swift

```diff
@@ -30,17 +30,17 @@ public final class ReferralChartView: UIView {
 
     guard let context = UIGraphicsGetCurrentContext() else { return }
 
-    let internalPercentageAngle = CGFloat(-M_PI_2) + self.internalPercentage * CGFloat(2.0 * M_PI)
+    let internalPercentageAngle = CGFloat(-.pi / 2.0) + self.internalPercentage * CGFloat(2.0 * .pi)
```

Comment:
this is a fun one! `pi` has been moved to a static var on `Double`, and the type-inference makes it really niiiiice